Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM ruby:2.5.1-alpine3.7
+RUN apk add --update --no-cache build-base libcurl \
+ && gem install github-pages-health-check \
```

Comment:
In this case, it would be most helpful if we bundled the repository instead of running `gem install`. Better for testing.